src/client/systems/sysplanetviewer.cpp

```diff
@@ -73,12 +73,15 @@ void SysPlanetInformation::DisplayPlanet() {
     } else {
         ImGui::SetNextWindowCollapsed(false, ImGuiCond_Always);
     }
-    std::string planet_name = "Planet";
+    std::string planet_name = "NULL";
     if (selected_planet == entt::null) {
         return;
     }
-    if (GetUniverse().all_of<cqspc::Name>(selected_planet)) {
-        planet_name = GetUniverse().get<cqspc::Name>(selected_planet);
+
+    if (GetUniverse().all_of<cqspc::Identifier>(selected_planet)) {
+        if (view_mode == ViewMode::PLANET_VIEW)
```

Comment:
You can use `cqsp::client::systems::gui::GetName()` from `src/client/systems/gui/systooltips.h`